docs/docs/rest-api/public/api/v2/shield.raml

```diff
@@ -49,9 +49,15 @@ description: |
       200:
         description: |
           The shield was successfully created or updated.
-      503:
+      400:
         description: |
-          Health check shield API is disabled.
```

Comment:
Just a question: why did you remove 503 return code? I would expect it to still be returned if the feature is disabled?

Reply:
Ah ok nvm, you merged the forbidden because of feature is disabled with forbidden because user does not have the rights to shield.